plugins/org.jkiss.dbeaver.model.jdbc/src/org/jkiss/dbeaver/model/impl/jdbc/struct/JDBCTable.java

```diff
@@ -182,18 +182,23 @@ public DBCStatistics readData(
         statistics.setQueryText(sqlQuery);
 
         monitor.subTask(ModelMessages.model_jdbc_fetch_table_data);
+        DBCStatement dbStat = null;
```

Comment:
Why changing code in JDBCTable?
We have zillion places where we execute statements.